lumicks/pylake/force_calibration/power_spectrum_calibration.py

```diff
@@ -87,8 +87,8 @@ def __getitem__(self, item):
 
     def plot(self):
         """Plot the fitted spectrum"""
-        self.ps_model_fit.plot()
         self.ps_fitted.plot()
+        self.ps_model_fit.plot()
         plt.legend(["Data", "Model"])
```

Comment:
Whould this not work and avoid the issue from happening again?
```suggestion
        self.ps_fitted.plot(title="Data")
        self.ps_model_fit.plot(title="Model")
```

Reply:
Fair point. I think it's called `label`, but yes, I reckon that'd be safer.

I've added it and argument forwarding in the `PowerSpectrum` plot function now.